documentation/certificates.md

```diff
@@ -2,24 +2,44 @@ Streisand PKI
 =============
 
 Streisand includes a role responsible for PKI certificate generation,
-separated into two distinct tasks:
+separated into three heirarchical tasks:
 
- - CA/Server
- - Client(s)
+```
+CA/Server
+   \__ CA Certificate
+     \__ Server certificate
+    \__ Client (1..n)
+      \__ Client PKCS (1..n)
+ 
+```
 
-The first will generate a certificate authority, and a server 
+CA/Server
+---------
+Guarded by `generate_ca_server: yes`
+
+Generate a certificate authority, and a server 
 certificate signed by the newly minted certificate authority.
 
-The latter task is responsible for generating client side certificates
-(using the same CA) used for authenticating connecting VPN clients.
+Client
+------
+Guarded by `generate_client: yes`
+
+Generate client side certificates using the same CA used for authenticating
+connecting VPN clients.
 
-Each VPN service that leverages the Streisand PKI generates its
-own CA/Server certificate and client certificates.
+Client PKCS
+-----------
+Guarded by `generate_pkcs: yes`
+
+Generate client PKCS files.
```

Comment:
Maybe a more descriptive comment here is: "Also convert client certificates into PKCS#12 format". They aren't distinct client certificates and "PKCS" without specifying "PKCS#12" is a little bit too vague.